packages/amplify-migration-tests/src/migration-helpers/init.ts

```diff
@@ -0,0 +1,72 @@
+import { addCircleCITags, getCLIPath, isCI, nspawn as spawn } from 'amplify-e2e-core';
+
+const defaultSettings = {
+  name: '\r',
+  envName: 'integtest',
+  editor: '\r',
+  appType: '\r',
+  framework: '\r',
+  srcDir: '\r',
+  distDir: '\r',
+  buildCmd: '\r',
+  startCmd: '\r',
+  useProfile: '\r',
+  profileName: '\r',
+  region: process.env.CLI_REGION,
+  local: false,
+  disableAmplifyAppCreation: true,
+};
+
+export function initJSProjectWithProfile(cwd: string, settings: Object, testingWithLatestCodebase = false) {
+  const s = { ...defaultSettings, ...settings };
+  let env;
+
+  if (s.disableAmplifyAppCreation === true) {
+    env = {
+      CLI_DEV_INTERNAL_DISABLE_AMPLIFY_APP_CREATION: '1',
+    };
+  }
+
+  addCircleCITags(cwd);
+
+  return new Promise((resolve, reject) => {
+    const chain = spawn(getCLIPath(testingWithLatestCodebase), ['init'], { cwd, stripColors: true, env })
+      .wait('Enter a name for the project')
+      .sendLine(s.name)
+      .wait('Enter a name for the environment')
+      .sendLine(s.envName)
+      .wait('Choose your default editor:')
+      .sendLine(s.editor)
+      .wait("Choose the type of app that you're building")
+      .sendLine(s.appType)
+      .wait('What javascript framework are you using')
+      .sendLine(s.framework)
+      .wait('Source Directory Path:')
+      .sendLine(s.srcDir)
+      .wait('Distribution Directory Path:')
+      .sendLine(s.distDir)
+      .wait('Build Command:')
+      .sendLine(s.buildCmd)
+      .wait('Start Command:')
+      .sendCarriageReturn()
+      .wait('Using default provider  awscloudformation');
+
+    if (testingWithLatestCodebase || !isCI()) {
```

Comment:
We could use the same check used by `getCLIPath()` to be more consistent.

Reply:
`getCLIPath()` performs the isCI check internally, this is to make sure the behavior matches as expected